### ISSUE TITLE
Contents of Video Search and Image Search do not get modified according to theme.

### ISSUE THREAD

**Issue by praveenojha33** [fossasia/susper.com#920]
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**
Contents of video search and Image search like Links,Titles,Description etc do not change its color according to theme.Further the links in "Did you mean:" also do not change.

**Expected behavior:**
It should change its color according to theme selected by the user.

**Screenshots of the issue:**
![err](https://user-images.githubusercontent.com/25537606/36808350-d566bdb0-1cea-11e8-8c7b-ccdb79d35700.png)
![err2](https://user-images.githubusercontent.com/25537606/36808456-230d0560-1ceb-11e8-974e-346d79c550a4.png)

**Comment by ValentinBercaru** [comment]
I added a commit and tested it and now I think it works fine. 
Surge link: https://pr-918-fossasia-susper.surge.sh

**Comment by praveenojha33** [comment]
@ValentinBercaru Please modify the color of auto-corrected links.

**Comment by ValentinBercaru** [comment]
@praveenojha33 I tried to edit the colors of the auto-correct part and I submitted a commit, but it seems that I do something wrong.
Here are the files I modified to change those colors: [Click to see the commit](https://github.com/fossasia/susper.com/pull/918/commits/d7167d90325ffc7e60759e8db79df9dfe2d173d6)

Travis CI returned the error `The command "ng test --single-run --code-coverage --reporters=coverage-istanbul" exited with 1.` [more info here](https://travis-ci.org/fossasia/susper.com/builds/347715474?utm_source=github_status&utm_medium=notification) 
And I don't know how to solve it so I added a commit with the preious version (without modified auto-correct colors). Any help?
